conventions/src/main/kotlin/otel.java-conventions.gradle.kts

```diff
@@ -92,7 +92,10 @@ abstract class NettyAlignmentRule : ComponentMetadataRule {
 dependencies {
   add(dependencyManagementConf.name, platform(project(":dependencyManagement")))
 
-  components.all<NettyAlignmentRule>()
+  // ratpack tests don't work correctly with 4.1.65.Final
```

Comment:
is this a separate flaky issue?

Reply:
The problem is that besides the fix that hopefully gets rid of the flakiness there is another change that breaks some tests when run with netty 4.1.65. These tests pass when run with the netty version that ratpack depends on. All tests in `RatpackPooledHttpClientTest` fail, from other http client tests only `highConcurrencyOnSingleConnection` fails, there are also failures under `library` tests.